web/packages/extension/assets/css/common.css

```diff
@@ -79,6 +79,10 @@ button {
     text-shadow: 0 0 0.1px white;
 }
 
+button:hover {
+    background: var(--ruffle-blue);
```

Comment:
That unfortunately doesn't look good in some places, because the hovered button has the same color as the background and it disappears, e.g.

![image](https://github.com/user-attachments/assets/9487abaa-70ef-48a8-a48b-0022fe6f0656)

I would suggest the following:

```suggestion
    filter: brightness(1.2);
```

![image](https://github.com/user-attachments/assets/0527fc9b-b1b9-4a9d-98ea-ea76264ba591)

Reply:
Thanks for the notice, I actually just intended this for the modal buttons.
I updated it to account for these buttons as well.